provisioning/tasks/init-redhat.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Get software for Python-based control.
+  yum: "name={{ item }} state=installed"
+  with_items:
+    - "curl"
+    - "python-pycurl"
+    - "@development"
+
+- name: Install postfix so Drupal can send emails.
+  yum: name=postfix state=installed
+
+# Use remi yum repository for PHP.
```

Comment:
I have a `geerlingguy.repo-remi` role to install the remi repo; so I'd prefer to just add that in the main playbook conditionally, instead of adding the extra steps here.